ui/raidboss/data/04-sb/raid/o3n.js

```diff
@@ -0,0 +1,474 @@
+        if (data.phaseNumber == 2)
+          return false;
+
+        data.holyTargets = data.holyTargets || [];
+        data.holyTargets.push(matches.target);
+        return data.holyTargets.length == 3;
+      },
+      alertText: function(data) {
+        if (data.holyTargets[0] == data.me) {
+          return {
+            en: 'Stack on YOU',
+            de: 'Stack auf DIR',
+          };
+        }
+        for (let i = 1; i < 3; i++) {
+          if (data.holyTargets[i] == data.me) {
+            return {
+              en: 'Get out',
+              de: 'Raus da',
+            };
+          }
+        }
+        return {
+          en: 'Stack on ' + data.holyTargets[0],
+          de: 'Stack auf ' + data.holyTargets[0],
+        };
+      },
+      run: function(data) {
+        delete data.holyTargets;
+      },
+    },
+    {
+      id: 'O3N Spellblade Holy Cave',
+      regex: Regexes.headMarker({ id: '0065' }),
+      condition: function(data, matches) {
+        return data.phaseNumber == 2 && data.me == matches.target;
+      },
+      response: Responses.spread(),
+    },
+    {
+      id: 'O3N Spellblade Holy Mindjack',
+      regex: Regexes.headMarker({ id: '0064' }),
+      condition: function(data) {
+        if (data.phaseNumber < 3)
+          return false;
+        data.holyCounter = data.holyCounter || 0;
+        return (data.holyCounter % 2 == 0);
+      },
+      response: Responses.stackOn(),
+      run: function(data) {
+        data.holyCounter += 1;
+        delete data.holyTargets;
+      },
+    },
+    {
+      id: 'O3N The Queen\'s Waltz: Crystal Square',
+      regex: Regexes.startsUsing({ id: '2471', source: 'Halicarnassus', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '2471', source: 'Halikarnassos', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '2471', source: 'Halicarnasse', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '2471', source: 'ハリカルナッソス', capture: false }),
+      regexCn: Regexes.startsUsing({ id: '2471', source: '哈利卡纳苏斯', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '2471', source: '할리카르나소스', capture: false }),
+      infoText: {
+        en: 'Get on crystal square',
+        de: 'Kristallfeld',
+      },
+      tts: {
+        en: 'blue square',
+        de: 'blaues feld',
+      },
+    },
+    {
+      id: 'O3N Great Dragon',
+      regex: Regexes.addedCombatant({ name: 'Great Dragon', capture: false }),
+      regexDe: Regexes.addedCombatant({ name: 'Riesendrache', capture: false }),
+      regexFr: Regexes.addedCombatant({ name: 'dragon suprême', capture: false }),
+      regexJa: Regexes.addedCombatant({ name: 'ドラゴングレイト', capture: false }),
+      regexCn: Regexes.addedCombatant({ name: '巨龙', capture: false }),
+      regexKo: Regexes.addedCombatant({ name: '거대 드래곤', capture: false }),
+      condition: function(data) {
+        return data.role == 'tank';
+      },
+      infoText: {
+        en: 'Grab dragon--point away',
```

Comment:
I do kind of feel like "point away" is implied for pretty much everything tanked.  It may just be enough to say "Dragon Add" for brevity.